tests/foreman/ui/test_errata.py

```diff
@@ -247,6 +254,82 @@ def test_end_to_end(session, module_repos_col, vm):
         assert result['result'] == 'success'
 
 
+@pytest.mark.tier2
+@pytest.mark.skipif((not settings.repos_hosting_url), reason='Missing repos_hosting_url')
+def test_content_host_errata_page_pagination(session, module_org, module_lce):
```

Comment:
This test is missing required testimony tokens, which is what's failing in CI:

```
2020-11-24T16:11:27.3394966Z testimony --config testimony.yaml validate tests/foreman/ui
2020-11-24T16:11:33.2431256Z tests/foreman/ui/test_errata.py
2020-11-24T16:11:33.2432205Z ===============================
2020-11-24T16:11:33.2432675Z 
2020-11-24T16:11:33.2433578Z test_content_host_errata_page_pagination:259
2020-11-24T16:11:33.2435476Z --------------------------------------------
2020-11-24T16:11:33.2436095Z 
2020-11-24T16:11:33.2437811Z * Docstring should have at least caseautomation, casecomponent, caseimportance, caselevel, id, requirement, testtype, upstream token(s)
2020-11-24T16:11:33.2439386Z 
```

Reply:
@mshriver Fixed now. I think my git hooks must have been broken in this branch as even the UUID was missing,  and that is normally checked by pre-commit hooks.